src/routes/open/components/SafeOwnersConfirmationsForm/index.tsx

```diff
@@ -201,7 +202,7 @@ const SafeOwnersForm = (props): React.ReactElement => {
                   placeholder="Owner Name*"
                   text="Owner Name"
                   type="text"
-                  validate={composeValidators(required, minMaxLength(1, 50))}
+                  validate={composeValidators(required, minMaxLength(1, 50), validAddressBookName)}
```

Comment:
This will force to set a name to all owners when creating a Safe

Reply:
removed